xcdat/bounds.py

```diff
@@ -1037,6 +1043,14 @@ def get_bounds_dim(da_bounds: xr.DataArray) -> str:
     This function is designed to work with DataArrays that include a dimension
     representing bounds, which is commonly used in geospatial or temporal data.
     """
+    # If only one bounds dimension is found, return it.
+    bounds_dim = [dim for dim in da_bounds.dims if dim not in da_coords.dims]
+
+    if len(bounds_dim) == 1:
+        return str(bounds_dim[0])
+
+    # FIXME: Do we still need this logic, or do we always expect bounds to have
+    # one additional dimension?
```

Comment:
For what it's worth – I looped over a bunch of CMIP data and never made into this if statement.